rotkehlchen/accounting/cost_basis/base.py

```diff
@@ -375,6 +379,16 @@ def consume_result(self, used_amount: FVal) -> None:
         `current_amount` is guaranteed to be greater than zero since `consume_result` is
         supposed to be called under `processing_iterator`.
         """
+        if self.current_amount == ZERO:
+            # this shouldn't happen but a user reported it in
+            # https://github.com/rotki/rotki/issues/7273. We couldn't find the reason for it so we
+            # decided to protect against it raising a error shown in the frontend
+            log.error(f'Division by zero error when processing report using ACB. {self._acquisitions_heap}')  # noqa: E501
+            raise AccountingError(
+                "Can't calculate the average cost basis because the remaining amount of the "
+                "asset is 0. Contact support and provide the log file for more information",
```

Comment:
Is this a user-facing message? Seems so. Do you really think this is informative? It gives half-details and those are even not clear. Why does he care about the remaining amount? What is that even? User does not know. Which asset?

 The user can't fix this on their own so they will anyway need to speak to us.
```suggestion
                "Remaining amount error during ACB calculation for {asset}. Contact support and provide the log file for more information",
```

To provide the {asset} it's possible you will need to add it to `BaseCostBasisMethod` though it may already be retrievable somehow. Not sure.